lib/xandra/cluster.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Xandra.Cluster do
+  @behaviour DBConnection.Pool
```

Comment:
`DBConnection.Pool` and `GenServer` don't have conflicting callbacks, so I would list them both here:

```elixir
use GenServer

@behaviour DBConnection.Pool
```